includes/class-reader-activation.php

```diff
@@ -597,7 +615,10 @@ public static function render_auth_form( $is_inline = false ) {
 							<?php _e( "We've recently sent you an authentication link. Please, check your inbox!", 'newspack' ); ?>
 						</p>
 						<p data-action="pwd">
-							<?php _e( 'Sign in below to verify your identity.', 'newspack' ); ?>
+							<?php \esc_html_e( 'Sign in below to verify your identity.', 'newspack' ); ?>
+						</p>
+						<p data-action="link">
+							<?php \esc_html_e( 'Get a link sent to your email address to sign in instantly without your password.', 'newspack' ); ?>
```

Comment:
"your" in this context implies that the user has a password, but this isn't always the case.

```suggestion
							<?php \esc_html_e( 'Get a link sent to your email address to sign in instantly without a password.', 'newspack' ); ?>
```